tosca-package/tosca/yaml2python.py

```diff
@@ -303,46 +303,57 @@ def find_repository(self, name) -> Tuple[str, str]:
             return name, url
         return name, name
 
-    def convert_import(self, tpl) -> Tuple[str, str]:
-        repository = tpl.get("repository")
-        import_path = os.path.dirname(self.template.path or "")
-        if repository:
-            module_name, import_path = self.find_repository(repository)
+
+
+    def convert_import(self, imp: dict[str, str]) -> Tuple[str, str]:
+        "converts tosca yaml import dict (as `imp`) to python import statement"
+        repo = imp.get("repository")
+        file = imp.get("file")
+        namespace = imp.get("namespace_prefix") or imp.get("namespace_uri") # _uri is deprecated
```

Comment:
namespace_uri isn't support by the tosca parser so this won't work anyway so might as well not mention